CONTRIBUTE.md

```diff
@@ -0,0 +1,49 @@
+# TIMRLINK CLI Datenbank Migrationen 
+​
+## timrlink.net.CLI.Migrations
+​
+Enthält das Datenmodell und den vollständigen Datenzugriff über Entity Framework Core.
+​
+### Vorbereitungen
+
+Zu Beginn müssen die dotnet ef tools installiert werden.
+
+https://learn.microsoft.com/en-gb/ef/core/cli/dotnet
+
+Das hab ich direkt in Rider als Paket (Nu Get) hinzugefügt (War einfacher).
```

Comment:
instead of protocoll what you have done, it would be nicer to have a guide for others, what they have to do to get it running